lib/recurly/base.php

```diff
@@ -399,6 +399,9 @@ protected static function __parseXmlToObject($node, &$object)
           // has element children, drop in and continue parsing
           $new_obj = Recurly_Resource::__createNodeObject($node);
```

Comment:
I wonder if it'd be better to just pass client into `__createNodeObject` as a parameter since most of the time we assign it afterwards. Then we'd avoid needing to check if it's an instance of Recurly_Base.

Reply:
Makes sense. I updated the pull request.